src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestServiceError.swift

```diff
@@ -34,7 +34,14 @@ enum CoronaTestServiceError: LocalizedError, Equatable {
 				return AppStrings.ExposureSubmissionError.noResponse
 			case .resourceError, .invalidResponse:
 				return AppStrings.ExposureSubmissionError.invalidResponse
-			case .invalidRequestError, .trustEvaluationError, .fakeResponse:
+			case .invalidRequestError:
+				Log.error("ExposureSubmissionError: serviceError invalidRequestError", log: .api)
```

Comment:
Wouldn't it make more sense/be cleaner to log where the error is occurring instead of when the localized description is accessed?

Reply:
mmm but how is that?, i think the bigger issue is that we are not sure what is causing the error and after the point of getting the localized generic message there is no way of knowing the exact cause, or am i missing something here ?